iOSClient/Viewer/NCViewerMedia/NCPlayer/NCPlayerToolBar.swift

```diff
@@ -359,7 +385,76 @@ class NCPlayerToolBar: UIView {
 
         timerAutoHide?.invalidate()
     }
-
+    
+    // MARK: - Message
+    
+    func showMessage(_ title: String, description: String?, backgroundColor: UIColor = NCBrandColor.shared.brand, isHiddenPregress: Bool = true, hiddenAfterSeconds: Double = 0) {
+        
+        self.playerMessageHeightConstraint.constant = 120
+        self.playerMessage.backgroundColor = backgroundColor
+
+        self.playerMessageTitle.text = NSLocalizedString(title, comment: "")
+        self.playerMessageTitle.textColor = NCBrandColor.shared.brandText
+        
+        self.playerMessageButton.isHidden = isHiddenPregress
+        self.playerMessageButton.setBackgroundImage(UIImage(named: "stop")!.image(color: .black, size: 30), for: .normal)
+        
+        if let description = description {
+            self.playerMessageDescription.isHidden = false
+            self.playerMessageDescription.text = NSLocalizedString(description, comment: "")
+            self.playerMessageTitleTopConstraint.constant = 8
+        } else {
+            self.playerMessageDescription.isHidden = true
+            self.playerMessageHeightConstraint.constant = 65
+            self.playerMessageDescription.text = ""
+            self.playerMessageTitleTopConstraint.constant = 20
+        }
+        self.playerMessageDescription.textColor = NCBrandColor.shared.brandText
+        
+        self.playerMessageProgressView.progress = 0
+        self.playerMessageProgressView.tintColor = .black
+        self.playerMessageProgressView.isHidden = isHiddenPregress
+        
+        self.playerMessageCommentBottomConstraint.constant = 30
+        if isHiddenPregress {
+            self.playerMessageCommentBottomConstraint.constant = 5
+            self.playerMessageHeightConstraint.constant = 90
+        }
+        
+        UIView.animate(withDuration: 0.3, animations: {
+            self.playerMessage.alpha = 1
+        }, completion: { (_: Bool) in
+            self.playerMessage.isHidden = false
+        })
+        
+        timerAutoHideMessage?.invalidate()
+        if hiddenAfterSeconds > 0 {
+            timerAutoHideMessage = Timer.scheduledTimer(timeInterval: hiddenAfterSeconds, target: self, selector: #selector(hideMessage), userInfo: nil, repeats: false)
+        }
+    }
+    
+    @objc func hideMessage() {
+        
+        self.playerMessageProgressView.progress = 0
+        
+        UIView.animate(withDuration: 0.3, animations: {
+            self.playerMessage.alpha = 0
+        }, completion: { (_: Bool) in
+            self.playerMessage.isHidden = true
+        })
+    }
+    
+    func showMessageProgress(_ progress: Float) {
+        
+        self.playerMessageProgressView.progress = progress
+    }
+    
+    func isHiddenMessage() -> Bool {
```

Comment:
Is this used anywhere? If not, please remove. If so, please so it's grammatically correct: `isMessageHidden`

Reply:
yes next